requirements.txt

```diff
@@ -1,3 +1,4 @@
+charachorder~=0.3.1
```

Comment:
This should use `charachorder.py`, no?

Reply:
Ah, yes. Actually, the package exists under both namespaces in PyPI. This is the older namespace which now serves as a mirror package. I'll update this with v1 when this PR will come closer to being merged.